packages/libpulseaudio/build.sh

```diff
@@ -28,25 +28,38 @@ termux_step_pre_configure () {
 termux_step_post_make_install () {
 	# Some binaries link against these:
 	cd $TERMUX_PREFIX/lib
-	for lib in pulseaudio/lib*.so* pulse-${TERMUX_PKG_VERSION}/modules/lib*.so*; do
+	for lib in pulseaudio/lib*.so pulse-${TERMUX_PKG_VERSION}/modules/lib*.so; do
 		ln -s -f $lib `basename $lib`
 	done
-	if [ $TERMUX_ARCH_BITS = "32" ]; then
-		SYSTEM_LIB=lib
-	else
-		SYSTEM_LIB=lib64
-	fi
+
 	# Pulseaudio fails to start when it cannot detect any sound hardware
 	# so disable hardware detection.
 	sed -i $TERMUX_PREFIX/etc/pulse/default.pa \
 		-e '/^load-module module-detect$/s/^/#/'
 	echo "load-module module-sles-sink" >> $TERMUX_PREFIX/etc/pulse/default.pa
-	cd $TERMUX_PREFIX/libexec
 
+	cd $TERMUX_PREFIX/bin
+	for bin in pamon paplay parec parecord; do
+		rm -f ../libexec/$bin
+		ln -s ../libexec/pacat $bin
```

Comment:
@tomty89 suppose to be `ln -s ../libexec/$bin $bin` here?

Reply:
Uhm no, it was supposed to be `ln -s ../bin/pacat $bin`. They were symlinks to `pacat` in `libexec/`. But I just notice that the `$0` magic doesn't work with a wrapper. Going to redo this.